app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/PrepareCopyTask.java

```diff
@@ -322,7 +327,12 @@ private void finishCopying(
     }
 
     if (filesToCopyPerFolder.size() != 0) {
-      int mode = mainActivity.mainActivityHelper.checkFolder(new File(path), context);
+      int mode = -1;
+      if (OpenMode.SMB.equals(openMode)) {
```

Comment:
The `checkFolder` is getting called from multiple places? This condition seem to be temporarily fixing only one of the places. My suggestion to use some form of abstraction or atleast make `checkFolder` generic to accept all paths and put this condition there.